examples/LXHeadless/src/heronarts/lx/headless/LXHeadless.java

```diff
@@ -22,10 +22,7 @@
 import heronarts.lx.LXPattern;
 import heronarts.lx.model.GridModel;
 import heronarts.lx.model.LXModel;
-import heronarts.lx.output.ArtNetDatagram;
-import heronarts.lx.output.FadecandyOutput;
-import heronarts.lx.output.LXDatagramOutput;
-import heronarts.lx.output.OPCOutput;
+import heronarts.lx.output.*;
```

Comment:
Are you using an IDE? Better to specifically enumerate only the dependencies that are used. Eclipse or IntelliJ should both do this...